charts/tfjob/templates/tfjob.yaml

```diff
@@ -22,22 +22,35 @@ metadata:
     {{ $key }}: {{ $value | quote }}
   {{- end }}
 spec:
-{{- if .Values.cleanPodPolicy }}
-{{- if eq "None" $cleanPodPolicy }}
+{{- if .Values.compatibleCRD }}
```

Comment:
Can it be called trainingOperatorCRD? It may be easy to understand for developers who don't have the background knowledge.

Reply:
agree, thank you!